srcpkgs/gummiboot/template

```diff
@@ -26,8 +27,10 @@ pre_configure() {
 }
 
 post_install() {
-	vinstall ${FILESDIR}/kernel.d/gummiboot.post-install 750 \
+	vinstall ${FILESDIR}/kernel.d/gummiboot.post-install 755 \
```

Comment:
I think it should be `744` instead. I.e. don't let unprivileged users touch it.

Reply:
Fixed